pkg/release/version.go

```diff
@@ -0,0 +1,15 @@
+package release
+
+type Version string
+
+func NewStableVersion() Version {
+	return Version(":stable")
+}
+
+func NewVersion(gitSHA string) Version {
+	return Version("@1.0.0-" + gitSHA)
```

Comment:
Uncertain if the @ is needed here.

Reply:
Sho how `helmclient` differentiates between channels and tags?

Reply:
PullChartTarballFromRelease

Reply:
Fixed.